From 9970d9433154511b29a9a73838d93c4ad46cda82 Mon Sep 17 00:00:00 2001
From: Duc Dang <dhdang@apm.com>
Date: Wed, 14 Oct 2015 17:24:29 -0700
Subject: [PATCH 20/27] arm64: dts: Add APM X-Gene v2 SoC GFC GPIO controller
 DTS entry

This patch adds the flash controller muxed gpio dts node for
APM X-Gene v2 SoC platforms.

Signed-off-by: Duc Dang <dhdang@apm.com>
---
 arch/arm64/boot/dts/apm/apm-shadowcat.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/apm/apm-shadowcat.dtsi b/arch/arm64/boot/dts/apm/apm-shadowcat.dtsi
index e7e6b4d..e32851e 100644
--- a/arch/arm64/boot/dts/apm/apm-shadowcat.dtsi
+++ b/arch/arm64/boot/dts/apm/apm-shadowcat.dtsi
@@ -424,6 +424,13 @@
 			clocks = <&sdioclk 0>, <&ahbclk 0>;
 		};
 
+		gfcgpio: gfcgpio@1f63c000 {
+			compatible = "apm,xgene-gpio";
+			reg = <0x0 0x1f63c000 0x0 0x40>;
+			gpio-controller;
+			#gpio-cells = <2>;
+		};
+
 		sbgpio: sbgpio@17001000{
 			compatible = "apm,xgene-gpio-sb";
 			reg = <0x0 0x17001000 0x0 0x400>;
-- 
2.6.2

